UI/MauiEmbedding/ArcGisApp/ArcGisApp/MainPage.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<Page x:Class="ArcGisApp.MainPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:ArcGisApp"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d"
+      xmlns:controls="using:ArcGisApp.MauiControls"
+      xmlns:uen="using:Uno.Extensions.Navigation.UI"
+      NavigationCacheMode="Required"
+      Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="Auto" />
+      <RowDefinition />
+    </Grid.RowDefinitions>
+    <TextBlock Text="{Binding Title}"
+               HorizontalAlignment="Center" />
```

Comment:
It can removed or add an actual Text, since we're not binding anything there.

Reply:
According to docs [(ArcGIS Maps SDK for .NET)](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Maui/ThirdParty-EsriMaps.html) it should be "Hello Uno Platform":

![image](https://github.com/unoplatform/Uno.Samples/assets/114886335/8d22fcef-8a01-498f-9c61-373e53ec1153)